### ISSUE TITLE
[Bug]: There are still errors when updating Debian

### ISSUE THREAD

**Issue by GitHubNeko** [termux/proot-distro#123]
### Problem description

I replaced https with the following command

sed -i -E 's@(http)s(://)@\1\2@g' /etc/apt/sources.list; apt update 

But when I update the source, it gives an error

404  Not Found [IP: 151.101.78.132 443]
Reading package lists... Done
E: The repository 'http://deb.debian.org/debian-security stable/updates Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
**Even if it can install the package, but the update source will always have this error**

### What steps will reproduce the bug?

apt update

**Comment by xeffyr** [comment]
Try these sources.list
```
deb http://deb.debian.org/debian buster main contrib
deb http://deb.debian.org/debian-security/ buster/updates main contrib
deb http://deb.debian.org/debian buster-updates main contrib
```

**Comment by GitHubNeko** [comment]
> 试试这个sources.list
> 
> ```
> deb http://deb.debian.org/debian buster main contrib
> deb http://deb.debian.org/debian-security/ buster/updates main contrib
> deb http://deb.debian.org/debian buster-updates main contrib
> ```

This sources is available 👍

**Comment by xeffyr** [comment]
Ok, fixed in 2.1.0.
